src/ansys/fluent/core/file_session.py

```diff
@@ -108,15 +108,15 @@ def add_scalar_fields_request(
 
         Raises
         ------
-        RuntimeError
+        TransactionError.SURFACE_NAMES_IDs_NOT_PROVIDED
             If surface names or surface ids are not provided.
```

Comment:
`TransactionError.SURFACE_NAMES_IDs_NOT_PROVIDED` is not a exception type, it can't be caught using `except`. 

I believe it makes more sense to list here just `TransactionError` as that is the exception type that is actually being raised and that can be caught.

Reply:
Until now, we are not catching exceptions, therefore we have not derived `TransactionError` from exception class.

Reply:
@raph-luc I just came across following test code.

![image](https://github.com/ansys/pyfluent/assets/106588300/8a0cc9a6-d6ca-422d-939a-85e0bbda2fdf)

Reply:
Thanks @hpohekar, that is an example of exactly what I am saying: you can't catch `TransactionError.SURFACE_NAMES_IDs_NOT_PROVIDED`, but you can catch `TransactionError`. 

Change your except block to `except Error` and it will work. In this example, it is therefore the `Error` type that should be listed in the numpy-style docstring, in my understanding.

Reply:
@raph-luc judging by @hpohekar 's other comments below, this issue should resolve itself shortly.